### ISSUE TITLE
No sound when trying to enable audio

### ISSUE THREAD

**Issue by Kinalvin** [rom1v/sndcpy#143]
Hello, here is what is currently displayed on my command prompt:

C:\Sndcpy> sndcpy
Waiting for device...
Performing Streamed Install
Success
Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=com.rom1v.sndcpy/.MainActivity }
Press Enter once audio capture is authorized on the device to start playing...
Playing audio...

However, no sound is playing on my desktop through sndcpy. The sound on my phone is playing just fine and even has a notification of "sndcpy: Audio forwarding enabled". VLC seems to be working for me (I deleted line 25 before in sndcpy's notepad and it only caused more issues). I tried apps like YouTube, the mobile game Fate/Grand Order, and so forth, and none of them were able to forward the phone's audio to my desktop. Furthermore, the audio on my desktop is working without any problems.

I am not familiar with coding, so I appreciate a solution to this problem. Thank you!

**Comment by kshitijsu** [comment]
I'm also facing the same issues with the sndcpy

**Comment by 1337Kamayuk** [comment]
I am experiencing a similar issue. However, my case is different. For me, sndcpy was working completely fine. Most apps worked properly. But now, none of them are working. I did update my phone's rom (using AEX custom rom on Poco f1, device code - beryllium). Now, after the update, nothing seems to be working fine. The only apps that are working fine are VLC player and Youtube. Other apps such as Poweramp music player are no longer working even though they were working fine before. I would greatly appreciate if anyone knows what must be done to fix this.

**Comment by PipGuy** [comment]
> Hello, here is what is currently displayed on my command prompt:
> 
> C:\Sndcpy> sndcpy
> Waiting for device...
> Performing Streamed Install
> Success
> Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=com.rom1v.sndcpy/.MainActivity }
> Press Enter once audio capture is authorized on the device to start playing...
> Playing audio...
> 
> However, no sound is playing on my desktop through sndcpy. The sound on my phone is playing just fine and even has a notification of "sndcpy: Audio forwarding enabled". VLC seems to be working for me (I deleted line 25 before in sndcpy's notepad and it only caused more issues). I tried apps like YouTube, the mobile game Fate/Grand Order, and so forth, and none of them were able to forward the phone's audio to my desktop. Furthermore, the audio on my desktop is working without any problems.
> 
> I am not familiar with coding, so I appreciate a solution to this problem. Thank you!

Does your phone become muted when the transmission happens? Maybe you can try to reset everything and start with a fresh sndcpy file

**Comment by Kinalvin** [comment]
> > Hello, here is what is currently displayed on my command prompt:
> > C:\Sndcpy> sndcpy
> > Waiting for device...
> > Performing Streamed Install
> > Success
> > Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=com.rom1v.sndcpy/.MainActivity }
> > Press Enter once audio capture is authorized on the device to start playing...
> > Playing audio...
> > However, no sound is playing on my desktop through sndcpy. The sound on my phone is playing just fine and even has a notification of "sndcpy: Audio forwarding enabled". VLC seems to be working for me (I deleted line 25 before in sndcpy's notepad and it only caused more issues). I tried apps like YouTube, the mobile game Fate/Grand Order, and so forth, and none of them were able to forward the phone's audio to my desktop. Furthermore, the audio on my desktop is working without any problems.
> > I am not familiar with coding, so I appreciate a solution to this problem. Thank you!
> 
> Does your phone become muted when the transmission happens? Maybe you can try to reset everything and start with a fresh sndcpy file

No, the phone still has sound during that process. I restarted the adb server and re-installed a new sndcpy file, but the issue remains unfortunately.

**Comment by stalker56** [comment]
> Hello, here is what is currently displayed on my command prompt:
> 
> C:\Sndcpy> sndcpy Waiting for device... Performing Streamed Install Success Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=com.rom1v.sndcpy/.MainActivity } Press Enter once audio capture is authorized on the device to start playing... Playing audio...
> 
> However, no sound is playing on my desktop through sndcpy. The sound on my phone is playing just fine and even has a notification of "sndcpy: Audio forwarding enabled". VLC seems to be working for me (I deleted line 25 before in sndcpy's notepad and it only caused more issues). I tried apps like YouTube, the mobile game Fate/Grand Order, and so forth, and none of them were able to forward the phone's audio to my desktop. Furthermore, the audio on my desktop is working without any problems.
> 
> I am not familiar with coding, so I appreciate a solution to this problem. Thank you!

Still no Solution for this?

**Comment by Narendhiran13** [comment]
I am also facing the same issue, upvote this if anyone faces the same issue!

**Comment by Tomiwa-Ot** [comment]
https://github.com/rom1v/sndcpy/issues/50#issuecomment-674956797

**Comment by bwanapete** [comment]
It seems there is nobody able to resolve this issue. I have watched people on YouTube resolve the problem but doing the same thing is not solving my issue. A FARCE!

**Comment by rom1v** [comment]
It probably depends on the device ROM which correctly exposes (or not) the sound via the Playback capture API.

**Comment by bwanapete** [comment]
I have made some kind of discovery?. I can get the Video on the laptop without sound or the Sound on the laptop without video. Now attempting to get both with little success.

**Comment by tomannyanomaly** [comment]
why my vlc cannot open it and it says "VLC media player could not start.
Either the command line option were invalid or no plugins were found"

idk what that mean

**Comment by Deepesh008** [comment]
I faced a similar issue, and I finally solved it. Open vlc first and check if its volume is at zero. If so, increase it.
